### PR TITLE
Various fixes to unit load paths, and systemd-analyze load-paths verb to list them

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/analyze/analyze.c [hunk 1]
```diff
@@ -294,7 +296,12 @@ static int acquire_boot_times(sd_bus *bus, struct boot_times **bt) {
                 return -EINPROGRESS;
         }
 
-        if (arg_user) {
+        if (arg_scope == UNIT_FILE_SYSTEM) {
+                if (times.initrd_time)
```
Comment (yuwata) [inline src/analyze/analyze.c]: coding style: `times.initrd_time > 0`

Comment (keszybz) [reply src/analyze/analyze.c]: That was existing code.

Comment (keszybz) [reply src/analyze/analyze.c]: Updated to fix this and similar style issues.

Comment (yuwata) [reply src/analyze/analyze.c]: Thank you and sorry for requesting such a cleanup.

File: man/systemd-analyze.xml [hunk 2]
```diff
@@ -188,6 +193,17 @@
     state. Its format is subject to change without notice and should
     not be parsed by applications.</para>
 
+    <para><command>systemd-analyze unit-paths</command> outputs a list of all
+    directories from which unit files, <filename>.d</filename> overrides, and
+    <filename>.wants</filename>, <filename>.requires</filename> symlinks may be
+    loaded. Combine with <option>--user</option> to retrieve the list for the user
+    manager instance, and <option>--global</option> for the global configuration of
+    user manager instances. Note that this verb prints the list that is compiled into
+    <command>systemd-analyze</command> itself, and does comunicate with the running
```
Comment (poettering) [inline man/systemd-analyze.xml]: "does communicate" → "does not communicate"?

Comment (keszybz) [reply man/systemd-analyze.xml]: Oops.

**Comment by keszybz** [comment]
Updated.

**Comment by keszybz** [comment]
```
test_no_options (__main__.JournalTest) ... Failed to process inotify events: Bad file descriptor
ERROR
```

**Pull request merged**